pkg/pgmodel/ingestor/dispatcher.go

```diff
@@ -58,6 +58,11 @@ func newPgxDispatcher(conn pgxconn.PgxConn, cache cache.MetricCache, scache cach
 	toCopiers := make(chan copyRequest, copierCap)
 	setCopierChannelToMonitor(toCopiers)
 
+	if cfg.IgnoreCompressedChunks {
+		// Handle decompression to not decompress anything.
+		handleDecompression = func(_ pgxconn.PgxConn, _ copyRequest) error { return nil }
```

Comment:
I think we should add a log here, to let the user know we are not ingesting old data. Which means this handler should be declared elsewhere, not inlined.